docs/asdf/deprecations.rst

```diff
@@ -81,5 +81,5 @@ include adding block storage support to :ref:`converters <extending_converters>`
 asdf.tests.helpers Deprecation
 ==============================
 
-Use of `asdf.tests.helpers` is deprecated. Please see `asdf.testing.helpers`
+Use of ``asdf.tests.helpers`` is deprecated. Please see `asdf.testing.helpers`
```

Comment:
I was thinking we'd leave this for at least 3.X as users migrating from <=2.15 might benefit from not needing to open older docs (2.15) to see what was deprecated/removed. It might end up being redundant if we can put together an exhaustive (but not exhausting) release doc for 3.0.